base/gfspapp/app_options.go

```diff
@@ -203,6 +224,85 @@ func DefaultGfSpDBOption(app *GfSpBaseApp, cfg *gfspconfig.GfSpConfig) error {
 	return nil
 }
 
+func DefaultGfBsDBOption(app *GfSpBaseApp, cfg *gfspconfig.GfSpConfig) error {
+	if val, ok := os.LookupEnv(BsDBUser); ok {
+		cfg.BsDB.User = val
+	}
+	if val, ok := os.LookupEnv(BsDBPasswd); ok {
+		cfg.BsDB.Passwd = val
+	}
+	if val, ok := os.LookupEnv(BsDBAddress); ok {
+		cfg.BsDB.Address = val
+	}
+	if val, ok := os.LookupEnv(BsDBDataBase); ok {
+		cfg.BsDB.Database = val
+	}
+	if val, ok := os.LookupEnv(BsDBSwitchedUser); ok {
+		cfg.BsDBBackup.User = val
+	}
+	if val, ok := os.LookupEnv(model.BsDBSwitchedPasswd); ok {
+		cfg.BsDBBackup.Passwd = val
+	}
+	if val, ok := os.LookupEnv(model.BsDBSwitchedAddress); ok {
+		cfg.BsDBBackup.Address = val
+	}
+	if val, ok := os.LookupEnv(model.BsDBSwitchedDataBase); ok {
+		cfg.BsDBBackup.Database = val
+	}
+
+	DefaultGfBsDB(&cfg.BsDB)
+	DefaultGfBsDB(&cfg.BsDBBackup)
+
+	metadataConfig := &metadata.MetadataConfig{
+		BsDBConfig:         &cfg.BsDB,
+		BsDBSwitchedConfig: &cfg.BsDBBackup,
+	}
+	bsDBBlockSyncerMaster, err := bsdb.NewBsDB(metadataConfig, false)
+	if err != nil {
+		return err
```

Comment:
if the process not start metadata and blocksyncer, and not config the bsdb seems the process  start.

Reply:
Fixed in https://github.com/bnb-chain/greenfield-storage-provider/pull/471